infra/sql/add_omop_source.sql

```diff
@@ -20,4 +20,82 @@ WHERE source_key = '${OMOP_CDM_DATABASE_NAME}';
 INSERT INTO ${WEBAPI_SCHEMA_NAME}.source_daimon (source_daimon_id, source_id, daimon_type, table_qualifier, priority) 
 SELECT nextval('${WEBAPI_SCHEMA_NAME}.source_daimon_sequence'), source_id, 5, '${POSTGRES_OMOP_TEMP_SCHEMA_NAME}', 0
 FROM ${WEBAPI_SCHEMA_NAME}.source
-WHERE source_key = '${OMOP_CDM_DATABASE_NAME}';
\ No newline at end of file
+WHERE source_key = '${OMOP_CDM_DATABASE_NAME}';
+
+-- todo: add source user role
+
+-- Add permissions for the OMOP CDM source
```

Comment:
Isn't this something ATLAS database seed/migration will do?

I think you have convert all this block to something like this and save half the rows:
```sql
INSERT INTO table_name (column_list)
VALUES
    (value_list_1),
    (value_list_2),
    ...
    (value_list_n);
```

Reply:
It's something that WebAPI does when you post new source (can also be done via Atlas UI), 
that's why I've marked out running this sql file in the bash script,
I can remove this file and sort this out later on in #130 preferably via REST call to WebAPI.

Reply:
Ok